termwiz/src/cell.rs

```diff
@@ -532,6 +532,42 @@ impl CellAttributes {
             .map(|fat| fat.underline_color)
             .unwrap_or(ColorAttribute::Default)
     }
+
+    pub fn apply(&mut self, change: &AttributeChange) {
```

Comment:
nit: naming wise, apply is very generic, so I'd suggest renaming to something like:

```suggestion
    pub fn apply_change(&mut self, change: &AttributeChange) {
```